web-test.runner-config.mjs

```diff
@@ -0,0 +1,49 @@
+import { esbuildPlugin } from '@web/dev-server-esbuild';
+import fg from 'fast-glob';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+
+export default {
+  rootDir: '.',
+  files: 'src/**/*.test.ts', // "default" group
+  concurrentBrowsers: 3,
+  nodeResolve: {
+    exportConditions: ['production', 'default'],
+  },
+  testFramework: {
+    config: {
+      timeout: 3000,
+      retries: 1,
+    },
+  },
+  plugins: [
+    esbuildPlugin({
+      ts: true,
+      target: 'es2020',
+    }),
+  ],
+  browsers: [
+    playwrightLauncher({ product: 'chromium' }),
```

Comment:
Sure, i'm currently installing the browsers with the default behavior but I'll update to install a specific version. Web test runner isn't defaulting to that config file, i set it up like that in the packageJson since there were no package specific configs i figured it would be better to just have one config at the root.